packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -5,7 +5,6 @@
     import { activeProfile } from '@core/profile'
     import { getNetwork, NetworkType, nodeInfo } from '@core/network'
     import { IAccountState } from '@core/account'
-
```

Comment:
This was probably an auto formatting issue but we should keep the empty line here for better readability.

Reply:
Done